development/eks-d.tpl

```diff
@@ -0,0 +1,143 @@
+  authorization:
+    rbac: {}
+  channel: stable
+  cloudProvider: aws
+  configBase: {{ .configBase }}
+  containerRuntime: docker
+  etcdClusters:
+  - cpuRequest: 200m
+    etcdMembers:
+    - instanceGroup: master-us-west-2a
+      name: a
+    memoryRequest: 100Mi
+    name: main
+  - cpuRequest: 100m
+    etcdMembers:
+    - instanceGroup: master-us-west-2a
+      name: a
+    memoryRequest: 100Mi
+    name: events
+  iam:
+    allowContainerRegistry: true
+    legacy: false
+  kubelet:
+    anonymousAuth: false
+  kubernetesApiAccess:
+  - 0.0.0.0/0
+  kubernetesVersion: {{ .kubernetesVersion }}
+  masterPublicName: api.{{ .clusterName }}
+  networkCIDR: 172.20.0.0/16
+  networking:
+    kubenet: {}
+  nonMasqueradeCIDR: 100.64.0.0/10
+  sshAccess:
+  - 0.0.0.0/0
+  subnets:
+  - cidr: 172.20.32.0/19
+    name: us-west-2a
+    type: Public
+    zone: us-west-2a
+  - cidr: 172.20.64.0/19
+    name: us-west-2b
+    type: Public
+    zone: us-west-2b
+  - cidr: 172.20.96.0/19
+    name: us-west-2c
+    type: Public
+    zone: us-west-2c
+  topology:
+    dns:
+      type: Public
+    masters: public
+    nodes: public
+  kubeAPIServer:
+    image: public.ecr.aws/eks-distro/kubernetes/kube-apiserver:v1.18.9-eks-1-18-1
+  kubeControllerManager:
+    image: public.ecr.aws/eks-distro/kubernetes/kube-controller-manager:v1.18.9-eks-1-18-1
+  kubeScheduler:
+    image: public.ecr.aws/eks-distro/kubernetes/kube-scheduler:v1.18.9-eks-1-18-1
+  kubeProxy:
+    image: public.ecr.aws/eks-distro/kubernetes/kube-proxy:v1.18.9-eks-1-18-1
+  # Metrics Server will be supported with kops 1.19
+  metricsServer:
+    enabled: true
+    image: public.ecr.aws/eks-distro/kubernetes-sigs/metrics-server:v0.4.0-eks-1-18-1
+  authentication:
+    aws:
+      image: public.ecr.aws/eks-distro/kubernetes-sigs/aws-iam-authenticator:v0.5.2-eks-1-18-1
+  kubeDNS:
+    provider: CoreDNS
+    coreDNSImage: public.ecr.aws/eks-distro/coredns/coredns:v1.7.0-eks-1-18-1
+    externalCoreFile: |
+      .:53 {
+          errors
+          health {
+            lameduck 5s
+          }
+          kubernetes cluster.local. in-addr.arpa ip6.arpa {
+            pods insecure
+            #upstream
+            fallthrough in-addr.arpa ip6.arpa
+          }
+          prometheus :9153
+          forward . /etc/resolv.conf
+          loop
+          cache 30
+          loadbalance
+          reload
+      }
+  masterKubelet:
+    podInfraContainerImage: public.ecr.aws/eks-distro/kubernetes/pause:v1.18.9-eks-1-18-1
+  # kubelet might already be defined, append the following config
+  kubelet:
+    podInfraContainerImage: public.ecr.aws/eks-distro/kubernetes/pause:v1.18.9-eks-1-18-1
+
+---
+
+apiVersion: kops.k8s.io/v1alpha2
+kind: InstanceGroup
+metadata:
+  creationTimestamp: null
+  labels:
+    kops.k8s.io/cluster: {{.clusterName}}
+  name: master-us-west-2a
+spec:
+  image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20201026
+  machineType: m5.large
```

Comment:
Lets reduce the instance type, no sense in making people pay for something this expensive in a demo (I had originally set it to this for testing)
```suggestion
  machineType: t3.medium
```